types/elm/tsconfig.json

```diff
@@ -5,10 +5,8 @@
             "es6",
             "dom"
         ],
-        "noImplicitAny": true,
-        "noImplicitThis": true,
-        "strictNullChecks": true,
-        "strictFunctionTypes": true,
+        "strict": true,
+        "exactOptionalPropertyTypes": true,
```

Comment:
My understanding was that this was acceptable, and I wanted to ensure it worked with exactOptionalProperty types (b/c I use it)